docs/mint.json

```
@@ -230,11 +237,30 @@
             "api-ref/providers/get-installed-providers",
             "api-ref/providers/install-provider-oauth2",
             "api-ref/providers/invoke-provider-method",
-
+            "api-ref/providers/get-alert-count",
+            "api-ref/providers/export-providers",
             "api-ref/providers/update-provider",
             "api-ref/providers/validate-provider-scopes"
           ]
         },
+
+        {
+          "group": "Dashboard",
+          "pages": [
+            "api-ref/dashboard/create-dashboard",
+            "api-ref/dashboard/delete-dashboard",
+            "api-ref/dashboard/read-dashboards",
+            "api-ref/dashboard/update-dashboard"
```

```suggestion
            "api-ref/dashboard/create-dashboard",
            "api-ref/dashboard/read-dashboards",
            "api-ref/dashboard/update-dashboard"
            "api-ref/dashboard/delete-dashboard",
```

CRUD? ;) Not important, comment just for fun =)

---

Non-intentionally :)